interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Controller/EncountermanagerController.php

```diff
@@ -56,7 +56,7 @@ public function indexAction()
         $status = $request->getPost('form_status', null);
 
         if (!$pid && !$encounter && !$status) {
-            $fromDate = $request->getPost('form_date_from', null) ? $this->CommonPlugin()->date_format($request->getPost('form_date_from', null), 'yyyy-mm-dd', $GLOBALS['date_display_format']) : date('Y-m-d', strtotime(date('Ymd')) - (86400 * 7 * 52 * 3));
+            $fromDate = $request->getPost('form_date_from', null) ? $this->CommonPlugin()->date_format($request->getPost('form_date_from', null), 'yyyy-mm-dd', $GLOBALS['date_display_format']) : date('Y-m-d', strtotime(date('Ymd')) - (86400 * 7 * 52 * 8)); // TODO give wide 8 year range for testing
             $toDate = $request->getPost('form_date_to', null) ? $this->CommonPlugin()->date_format($request->getPost('form_date_to', null), 'yyyy-mm-dd', $GLOBALS['date_display_format']) : date('Y-m-d', strtotime('2022-12-31')); // TODO remove date adjustment production
```

Comment:
@bradymiller Can you give me a hint on how to match local? I'm befuddled...

Reply:
hi @sjpadgett , with the standard date picker widget, gotta do 3 things:
1. widget itself is set to support the local setting: https://github.com/openemr/openemr/blob/master/interface/reports/appointments_report.php#L141
which is used in the js for standard date picker widget at
https://github.com/openemr/openemr/blob/master/library/js/xl/jquery-datetimepicker-2-5-4.js.php#L104
(this will ensure when select a date in the widget js will populate the field with the local setting)

2. when directly populate the field with a default value (will convert it from standard (yyyy-mm-dd) to the local setting):
https://github.com/openemr/openemr/blob/master/interface/reports/appointments_report.php#L247

3. when collecting the value from the request parameter (convert it from local setting to standard):
https://github.com/openemr/openemr/blob/master/interface/reports/appointments_report.php#L61
 
will be much easier to change over to use openemr standard date widgets or else number 1 above will likely make things very complicated

Reply:
i'd also drop use of the commonPlugin functions and just use the functions used in above examples directly

Reply:
Yep I missed this is a diff picker. Thanks. I'll bring this in and do on separate PR so Stephen can have for testing tomorrow.